pytoml/writer.py

```diff
@@ -87,10 +90,19 @@ def _format_value(v):
             return v.strftime('%Y-%m-%dT%H:%M:%S') + suffix
     elif isinstance(v, list):
         return _format_list(v)
+    elif _isinstance_purepath(v):
+        return '"{}"'.format(v)
     else:
         raise RuntimeError(v)
 
 
+def _isinstance_purepath(v):
```

Comment:
You can completely drop this if you incorporate the `_path_types` suggestion above.